src/pages/DoctorDashboard.tsx

```diff
@@ -112,15 +139,15 @@ function DoctorDashboard() {
             </Container>
 
             <Container>
-                <Col className={`d-flex${Visits.length === 0 ? '' : '-nowrap'} justify-content-center`}>{Visits.length === 0 ? (loading ? <Spinner animation='border'/> : <div>{error}</div>) : Visits.map((field, index) => {
+                <Col className={`d-flex${Visits.length === 0  && !loading ? '' : '-nowrap'} justify-content-center`}>{loading ?  <Spinner animation='border'/> : (Visits.length===0 ?  <div>{error}</div> : Visits.map((field, index) => {
```

Comment:
<Container>
                <Col
                    className={`d-flex${Visits.length === 0 && !loading ? '' : '-nowrap'} justify-content-center`}
                >
                    {loading
                        ? <Spinner animation='border'/>
                        : (Visits.length === 0
                            ? <div>{error}</div>
                            : Visits.map((field, index) => {
                                return <PatientVisitField
                                    key={`visit_${index}`}
                                    visit={field} index={index}
                                    remove={remove}
                                />
                            }))
                    }
                </Col>
            </Container>
```suggestion
                <Col className={`d-flex${Visits.length === 0  && !loading ? '' : '-nowrap'} justify-content-center`}>{loading ?  <Spinner animation='border'/> : (Visits.length===0 ?  <div>{error}</div> : Visits.map((field, index) => {
```

Reply:
Może jakoś tak jak wyżej?